package.json

```diff
@@ -23,8 +23,8 @@
         "prepare": "husky install",
         "deprecate-old-versions": "node scripts/deprecate-old-versions.mjs"
     },
-    "main": "dist/module.js",
-    "module": "dist/es.js",
+    "main": "dist/main.js",
+    "module": "dist/module.js",
```

Comment:
ELI5? 

(i have managed to ignore changes to JS bundling over the last few chaotic years 🤣)

Reply:
I think we just had it poorly named beforehand. The module wasn't really the module loader 🤷